src/hardware/driver/mod.rs

```diff
@@ -61,7 +61,8 @@ impl ChannelVariant {
 pub enum Reason {
     #[default]
     Reset, // Tripped after device reset
-    Thermostat,
+    ThermostatLimits,
```

Comment:
This isn't specific to `Thermostat` or `Limits`. Just call the two variants `Alarm` and `AlarmTimeout`.